torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -2165,36 +2160,6 @@ void InitXlaModuleBindings(py::module m) {
         ShardingUtil::CreateShardedData(shards, devices, sharding_spec);
     xtensor->SetXlaData(xla_data);
   });
-  // This is useful for debugging and generating a partitioned HLO separately
-  // outside the actual compilation & execution. This allows testing with
-  // different partitioning configurations.
-  m.def("_xla_partitioning_pass",
```

Comment:
I will try to break this out to a separate PR for a quick review, https://github.com/pytorch/xla/pull/6720